<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barchartClass</class>
 <widget class="QWidget" name="barchartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>barchart</string>
  </property>
  <widget class="QSpinBox" name="inputspinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputspinBox2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="outputWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="barchart.qrc"/>
 </resources>
 <connections/>
</ui>
